documentation/book/con-resource-kafka-connect.adoc

```diff
@@ -0,0 +1,280 @@
+[id='kafka-connect-resource-{context}']
+[[kafka_connect_config_map_details]]
+= Kafka Connect
+
+The operator watches for `KafkaConnect` resource in order to find and get configuration for a Kafka Connect cluster to deploy.
+
+The `KafkaConnectS2I` resource provides configuration for a Kafka Connect cluster deployment using Build and Source2Image features (works only with {OpenShiftName})
```

Comment:
I'd add an ifdef around the parens here since we're only talking about OpenShift in the product.